doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -701,6 +703,7 @@ The Matter fork in the |NCS| (``sdk-connectedhomeip``) contains all commits from
 The following list summarizes the most important changes inherited from the upstream Matter:
 
 * Added the Binding cluster and Groupcast communication to the Light Switch sample.
+* Added for nRF Connect SDK platform :file:`Kconfig.defaults`, :file:`Kconfig.mcuboot.defaults` and :file:`Kconfig.multiprotocol_rpmsg.defaults` files containing default configuration common for all nRF Connect SDK samples.
```

Comment:
```suggestion
* Updated the nRF Connect platform by adding :file:`Kconfig.defaults`, :file:`Kconfig.mcuboot.defaults` and :file:`Kconfig.multiprotocol_rpmsg.defaults` files that contain the default configuration for all nRF Connect SDK samples.
```
The platform in Matter is called `nRF Connect` last time I checked, not `nRF Connect SDK`, but maybe you meant something else here than my suggestion? Let me know.